### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by adriencyberspace**
# Pull Request Template

## Issue Overview

This PR addresses [#91](https://www.notion.so/exygy/Footer-5b7c37dec020453db5601aae545c4ff5)
CMS PR (Sanity schema) here: https://github.com/Exygy/our415-cms/pull/4

- [X] This change addresses the issue in full
- [ ] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

This PR replaces the old footer with the new mockup version, as well as pulls the contact info, column titles and links from Sanity Studio. 

## How Can This Be Tested/Reviewed?

Add sanity env vars from `config.example.yml` into `config.yml` and add the project id var from sanity studio settings.

To review, visit any page and scroll down to the footer. 

Visual / A11y QA: 
- Should be correctly populated from Sanity Studio
- Internal links should open in the same tab (starting with '/'). External links should open in a different tab (all links not starting with '/'. There are a few placeholders with href '#').
- View on different sized screens
- Tab and screen reader navigate through links 

Sanity:
- You can edit the footer in the Our415 Development sanity project and the changes should reflect in the footer

No tests yet.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I have assigned reviewers
- [x] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "ShelterTech",
+  "name": "sheltertech",
```
Comment (adriencyberspace) [inline package.json:2]: FYI "ShelterTech" was causing a prettier error so I changed it

**Review by kramduckner — COMMENTED** [review]
Looks like a million bucks! Just added a couple comments. Could we also try to update as much of scss to use the scss tokens? Feel free to check out [this PR ](https://github.com/Exygy/askdarcel-web/pull/28) for examples

File: app/components/ui/Footer/Footer.scss [hunk 2]
```diff
@@ -0,0 +1,137 @@
+@import "../../../styles/utils/_helpers.scss";
+
+.site-footer {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  flex: 1 0 30vh;
```
[the remark this answers is not in the document]
Comment (adriencyberspace) [reply app/components/ui/Footer/Footer.scss:7]: Totally! I must have left that in from the first footer

**Comment by adriencyberspace** [comment]
@kramduckner I updated to include the existing spacing scss vars but I have a ticket to add more styling, at which point I can add other vars i.e. colors.

**Review by kramduckner — APPROVED** [review]
lookin good!

**Pull request merged**